### PR TITLE
Norfstar plushies get names

### PR DESCRIPTION
**Pull request by Zargoar**
## About The Pull Request
Adds names to a few stuffed toys to be found around Northstar.

## Why It's Good For The Game

Adding uniqueness to the plushies that are found around Northstar adds further character and charm to the map.
Adding this detail will make the station feel more lived-in.

## Changelog
qol: adds further flavor to the Northstar Plushies

### CODE REVIEW

**Comment by Cheshify** [comment]
I appreciate the names, I just don't get a lot of them I'm afraid.

**Comment by Zargoar** [comment]
> I appreciate the names, I just don't get a lot of them I'm afraid.

Bites-the-wires is just maintenance gremlin .
Congo Ralis is taken from Conogethes punctiferalis a species of moth featured in a research paper about mothic courtship songs.
Nitrous II refrences Nitrous Oxide
Bee Have = Behave

**Comment by Cheshify** [comment]
ahhh gotcha

File: _maps/map_files/NorthStar/north_star.dmm [hunk 1]
```diff
@@ -31413,7 +31413,10 @@
 /area/station/command/heads_quarters/cmo)
 "ihg" = (
 /obj/effect/decal/cleanable/dirt,
-/obj/item/toy/plush/lizard_plushie/green,
+/obj/item/toy/plush/lizard_plushie/green{
+	name = "Bites-The-Wires";
```
Comment (Iamgoofball) [inline _maps/map_files/NorthStar/north_star.dmm:31417]: no map edits, no player refereces

Comment (Zargoar) [reply _maps/map_files/NorthStar/north_star.dmm:31417]: > no map edits, no player refereces

Oh is that someone's static? I was just picking a generic Argonian name that would work for a maint lizard.
I can change that no sweat.

Comment (Iamgoofball) [reply _maps/map_files/NorthStar/north_star.dmm:31417]: oh, in that case just make these actual plushie subtypes in the code and place them we dont do var edited items on maps

Comment (Mothblocks) [reply _maps/map_files/NorthStar/north_star.dmm:31417]: Non-reused var edits are completely fine

Comment (Pumpkinoe) [reply _maps/map_files/NorthStar/north_star.dmm:31417]: >  we dont do var edited items on maps

tell that to every edited plushie on every map

**Pull request merged**